sda-commons-server-spring-data-mongo/src/main/java/org/sdase/commons/server/spring/data/mongo/SpringDataMongoBundle.java

```diff
@@ -227,6 +266,7 @@ private void registerOnShutdown(Environment environment) {
                   () -> {
                     applicationContext.stop();
                     applicationContext.close();
+                    mongoClient.close();
```

Comment:
Should we add this in a separate shutdown in case something goes wrong when shutting down the application context?
Actually, this won't be called when the applicationContext is null (= no validation enabled).

Reply:
You are right (again 😄 )